spec/integration_spec.rb

```diff
@@ -285,12 +285,14 @@ def test_unicode
   context "RSpec" do
     it_fails_without_any_files "rspec"
 
-    it "captures seed with random failures" do
-      write 'spec/xxx_spec.rb', 'describe("it"){it("should"){puts "TEST1"}}'
-      write 'spec/xxx2_spec.rb', 'describe("it"){it("should"){1.should == 2}}'
-      result = run_tests "spec", :add => "--test-options '--seed 1234'", :fail => true, :type => 'rspec'
-      expect(result).to include("Randomized with seed 1234")
-      expect(result).to include("bundle exec rspec spec/xxx2_spec.rb --seed 1234")
+    context 'with verbose option' do
```

Comment:
not a big fan of empty contexts ... 

how about `it "captures seed with random failures with --verbose" do` :)